deleteDiscordMessages.js

```diff
@@ -50,19 +53,35 @@
     const startBtn = popup.document.querySelector('button#start');
     const stopBtn = popup.document.querySelector('button#stop');
     const autoScroll = popup.document.querySelector('#autoScroll');
+    const fileSelection = popup.document.querySelector("input#file");
+    fileSelection.addEventListener("change", () => {
+        const files = fileSelection.files;
+        const channelIdField = popup.document.querySelector('input#channelId');
+        if (files.length > 0) {
+            const file = files[0];
+            file.text().then(text => {
+                let json = JSON.parse(text);
+                let channels = []
+                Object.keys(json).forEach(k => {
+                    channels.push(k);
+                })
+                channelIdField.value = channels.join(";");
+            });
+        }
+    }, false);
     startBtn.onclick = e => {
         const authToken = popup.document.querySelector('input#authToken').value.trim();
         const authorId = popup.document.querySelector('input#authorId').value.trim();
         const guildId = popup.document.querySelector('input#guildId').value.trim();
-        const channelId = popup.document.querySelector('input#channelId').value.trim();
+        const channelIds = popup.document.querySelector('input#channelId').value.trim().split(";");
```

Comment:
I think we can make this more user friendly by making this field more forgiving 

```suggestion
        const channelIds = popup.document.querySelector('input#channelId').value.split(/[;,]/).map(s=>s.trim()).filter(Boolean);
```

This will trim each id, separate by ; or , and remove empty positions
```js
> ",11111111 ;; 222222;3333; 44444;55555 ,6666, 77777,    8888".split(/[;,]/).map(s=>s.trim()).filter(Boolean)
["11111111", "222222", "3333", "44444", "55555", "6666", "77777", "8888"]
```